build-tools/packages/build-cli/tsconfig.json

```diff
@@ -1,13 +1,12 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
+	"include": ["src/**/*"],
```

Comment:
Shouldn't this still exclude src/test?

Reply:
All of the test code is under build-cli/test rather than build-cli/src/test; so the prior exclusion wasn't doing anytning.

Reply:
build-cli tests are not in the src folder. They're in a top-level test folder, a peer to src.